<launch>
	<rosparam command="load" file="$(find mongodb_api)/config/server_config.yaml"/>
	<node name='ros_mongo_server' pkg='mongodb_api' type='ros_mongo_server.py' output='screen'/>
</launch>
